### DISCUSSION TITLE
add it to the truecharts repository

### DISCUSSION THREAD

**Discussion by JosefAschauer** [music-assistant/support#1860] [Feature requests and ideas]
Hi, I'm running TrueNAS scale, which includes kubernetes. Wouldn't it be a good idea to add your docker image to the truecharts repository?

**Comment by OzGav** [comment]
If you want to maintain that then go for it. We don’t have capacity to do anymore than what we are already doing and we recommend people use HAOS anyway. Sorry.

**Comment by JosefAschauer** [comment]
All good. Thanks for repling anyways!

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________
